pkg/cloud/openstack/machineactuator.go

```diff
@@ -52,6 +56,8 @@ const (
 
 	TimeoutInstanceCreate       = 5 * time.Minute
 	RetryIntervalInstanceStatus = 10 * time.Second
+
+	TokenTTL = 30 * time.Minute
```

Comment:
30mins might not be enough in some scenarios where either network is not super fast or the nodes bootstrap process takes longer. Sounds horrible but it's possible :smile:

I'd go from 60 mins on.